tests/unit/testsuite/Bolt/Boltpay/Model/OrderTest.php

```diff
@@ -157,4 +157,195 @@ public function testActivateOrder()
         $this->assertEquals('TEST-BOLT-REFE-TWO2', $payment->getAdditionalInformation('bolt_reference'));
         ////////////////////////////////////////////////////////////
     }
+
+    /**
+     * @test
+     * That when the discount price is the same at Bolt and Magento, no exception is thrown
+     *
+     * @covers ::validateTotals
```

Comment:
@mccottry according documentation https://phpunit.de/manual/4.8/en/appendixes.annotations.html#appendixes.annotations.covers
@cover ::validateTotals specifies that the annotated test method covers the specified global function.

Reply:
@mccottry please also add group annotation - this allow in future run not all unit tests.
Hint: each method could belong to several groups, it make sense add such groups:
@group Model - in order to test only models
@group Order - to run tests for Orders
@group ModelOrder - to run tests from one file

Reply:
@oldDriver  use --filter